### DISCUSSION TITLE
Validation error for BT-634 Procurement relaunch in a CN against BT-142 Winner chosen, which is used only in CANs

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#406] [Q&A] [answered]
Hi,
I get a validation error when BT-634 Procurement relaunch is "true" in a contract notice (form 16). It validates against BT-142, which is only present in contract award notices. That needs to be corrected.

Kind regards,
Åsa  

![image-20230328-143241](https://user-images.githubusercontent.com/79446518/231187665-00c8d36c-82a7-4c8d-81b3-e7839221fd95.png)

**Comment by YvesJo** [comment, marked as answer]
Hi Åsa,
When the initial competition is published and the lots defined, there is no point to use BT-634; this indicator is only useful for a competition notice when this notice is changed with the Change Reason Code (BT-140) set to "The notice (or its specific Lot or Part) is cancelled".
When this helps, we may enrich for the next versions the set of rules so that BT-634 would only be accessible in the few cases where it is actually useful.
